gapic-common/lib/gapic/paged_enumerable.rb

```diff
@@ -52,11 +52,12 @@ class PagedEnumerable
     # @param method_name [Symbol] The RPC method name.
     # @param request [Object] The request object.
     # @param response [Object] The response object.
+    # @param operation [GRPC::ActiveCall::Operation] the current RPC operation.
```

Comment:
I think adding `operation` to this object is good. I think we could add additional `each_with_operation` and `each_page_with_operation` methods to make it easier for users to access this information.

Reply:
I added a `each_page_with_operation` but I'm not sure a `each_with_operation` really makes sense?